spec/oncocore_stage.txt

```diff
@@ -5,7 +5,7 @@ Uses:              shr.core, obf
 CodeSystem:        LNC = http://loinc.org
 
                    //---------- General definitions of Stage Group and Stage Category ------
-Abstract:          CancerStageGroup
+Entry:             CancerStageGroup
```

Comment:
Assuming this is the modeling fix for https://standardhealthrecord.atlassian.net/browse/MCODE-5.  If so, then great!...and note to self: fix all the DSTU2 and R4 FHIR examples involving CancerStagingGroup.